<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inductance_calculation</class>
 <widget class="QWidget" name="Inductance_calculation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_up">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Параметры</string>
   </property>
  </widget>
  <widget class="Line" name="line_up">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>439</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_down">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>439</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text_L">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>350</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="tabChangesFocus">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_Down">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Значение</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>377</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" rowspan="5">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" rowspan="5">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_N">
          <property name="text">
           <string>Количество витков (N)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_nu">
          <property name="text">
           <string>Магнитная проницаемость сердечника (µ)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_R">
          <property name="text">
           <string>Наружный радиус (R)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_r">
          <property name="text">
           <string>Внутренний радиус (r)</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_h">
          <property name="text">
           <string>Высота тороидального сердечника (h)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_N"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_nu"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_R"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_r"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_h"/>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_pieces">
      <property name="text">
       <string>шт</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_none">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_m_R">
      <property name="text">
       <string>м</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_m_r">
      <property name="text">
       <string>м</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_m_h">
      <property name="text">
       <string>м</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QPushButton" name="Clear_calculation">
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="L_calculation">
      <property name="text">
       <string>Расчёт L</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
